test/addons-abi/3_callbacks/binding.cc

```diff
@@ -41,8 +41,7 @@ void Init(napi_env env, napi_value exports, napi_value module) {
     { "RunCallback", RunCallback },
     { "RunCallbackWithRecv", RunCallbackWithRecv }
   };
-  status = napi_define_properties(
-    env, exports, sizeof(desc) / sizeof(napi_property_descriptor), desc);
+  status = napi_define_properties(env, exports, 2, desc);
```

Comment:
Ok, I buy that. Still would be nice for a convenience macro to make reading some of this code easier but I don't care too strongly